config/base.json

```diff
@@ -30,8 +30,9 @@
 		"max_static_paths": 10
 	},
 	"flags": {
-		"enable_product_docs_search": true,
+		"enable_auth": true,
```

Comment:
⛏️ I think the base state should be `false`, at least during this development phase. That way we're enabling auth explicitly in dev/preview as opposed to disabling in prod. Super small nit, so feel free to ignore if you don't vibe with this :)

Reply:
This makes sense to me! Updated in https://github.com/hashicorp/dev-portal/pull/739/commits/bda9e7e1dbfa7f03c3cdcceb7d004b065fbb29e7. 😊